sources/DropZoneTab.cpp

```diff
@@ -46,7 +51,7 @@ DropZone::_Init()
 
 DropZone::DropZone(bool replicatable)
 	:
-	BView("Filer dropzone", B_WILL_DRAW | B_FULL_UPDATE_ON_RESIZE),
+	BView(B_TRANSLATE("Filer dropzone"), B_WILL_DRAW | B_FULL_UPDATE_ON_RESIZE),
 	fReplicated(false)
```

Comment:
I purposefully skipped all internal names. I think this one is a BView label and remember it appears somewhere in the GUI. Will double check.